src/test/skript/tests/junit/InventoryMoveItemEventTest.sk

```diff
@@ -0,0 +1,22 @@
+
+options:
+	junit: org.skriptlang.skript.test.tests.syntaxes.InventoryMoveItemEventTest
+	objective: inventory move item event success
+
+on script load:
+	ensure junit test "{@junit}" completes "{@objective}"
+
+on inventory item move:
+	junit test is "{@junit}"
+
+	# Refers to getBlock() in SimpleJUnitTest.java
+	set {_location} to (spawn of first element of all worlds) ~~ vector(10, 1, 0)
+
+	block at {_location} is a chest
+	block below {_location} is a hopper
+	assert past event-block is a chest with "Past event-block should be a chest"
+	assert event-block is a hopper with "event-block should be a hopper"
+	assert event-initiator-inventory is a hopper with "event-initiator-inventory should be a hopper"
+	assert event-item is a stone with "event-item should be a stone"
+
+	complete objective "{@objective}" for junit test "{@junit}"
```

Comment:
```suggestion
	complete objective "{@objective}" for junit test "{@junit}"

```

Reply:
assuming the tests are fixed, looks good

Reply:
It is indeed failing from an illegal cast too — I tried to cast a Block to a Chest where Chest is a BlockState. Whoops.